wiki/entities/attributes/markdown/attributes.md

```diff
@@ -0,0 +1,207 @@
+
+> When creating your own entity classes, you can use `FabricDefaultAttributeRegistry.register` to take care of this without your own mixins.
+
+3. During play, the `LivingEntity` constructor initializes the `attributes` field as a new `AttributeContainer` instance based on the result of the getter in `DefaultAttributeRegistry` mentioned prior. This is the value returned in the `LivingEntity#getAttributes` getter.
+
+4. When *using* an attribute's value stored in an entity, you need to retrieve its corresponding `EntityAttributeInstance`; we'll go over this in more detail later. For now, all you should know is that **if the entity doesn't have an attribute in its `attributes` field, it won't have that attribute instance**, and you won't be able to apply a modifier.
+
+### Mixing Into Attribute Creation
+
+Each entity that does something different with attributes has their own attribute creation method, which you can find in `DefaultAttributeRegistry`. Since we want our attribute on all entities, we'll mix into the `LivingEntity.createLivingAttributes` method shown in the previous section.
+
+Remember, at this point, the builder is still immutable - and a reference - so our mixin is very straightforward:
+
+```java
+
+@Mixin(LivingEntity.class)
+public class LivingEntityMixin {
+
+    @Inject(method = "createLivingAttributes", at = @At(value = "RETURN"))
+    private static void addCustomAttribute(CallbackInfoReturnable<DefaultAttributeContainer.Builder> cir) {
+        cir.getReturnValue().add(ExampleMod.GENERIC_JUMP_BOOST);
+    }
+}
+```
+
+## Using Attributes
+
+We've registered our attribute and let all living entities know that they have it. The next step is to actually use it somewhere: in our case, to modify jump height.
+
+`LivingEntity#getAttributes` gets the *possible* attributes that the entity knows it has, but not the actual values. Each attribute has its own 'instance' on an entity known as an `EntityAttributeInstance`, which has all the logic for controlling attribute values. You can use the `LivingEntity#getAttributeInstance` method for getting an instance.
+
+We'll need another mixin into `LivingEntity`, this time for the `getJumpVelocity` method. Since our initial fallback value is `1.0`, we can feel free to just multiply the end result by the attribute value. Here's what that looks like:
+
+```java
+@Mixin(LivingEntity.class)
+public abstract class LivingEntityMixin {
+    // ...
+
+    @Shadow public @Nullable abstract EntityAttributeInstance getAttributeInstance(EntityAttribute attribute);
+
+    @Inject(method = "getJumpVelocity", at = @At(value = "RETURN"), cancellable = true)
+    private void applyJumpBoost(CallbackInfoReturnable<Float> cir) {
+        EntityAttributeInstance instance = getAttributeInstance(ExampleMod.GENERIC_JUMP_BOOST);
+        cir.setReturnValue(cir.getReturnValue() * (float) instance.getValue());
```

Comment:
Should the wiki advocate for unconditional cancels in situations like this?
Not that there is an alternative here without the use of 3rd party libraries, just a thought.